smoothing_frequency: 20.0
decel_factor: 1.0
max_vel_x: 0.7
max_vel_theta: 0.6
max_accel_x: 1.5
max_accel_theta: 2.0